src/en/index.md

```diff
@@ -627,6 +629,134 @@ func fetchAll() async {
 
 If you're already in an async context, prefer structured concurrency (`async let`, `TaskGroup`) over creating unstructured `Task`s. Structured concurrency handles cancellation automatically and makes the code easier to reason about.
 
+<div id="managedtasks">
+
+### Create unmanaged tasks
+
+Tasks that you create manually with `Task { ... }` or `Task.detached { ... }` are not managed. After you create unmanaged tasks, you can't control them. You can't cancel them if the task from which you started it is cancelled. You can't know if they finished their work, if they throw an error, or collect their return value. Starting such a task is like throwing a bottle into the sea and hoping it will deliver its message to its destination, without ever seeing that bottle again.
+
+<div class="analogy">
+<h4>The Office Building</h4>
+
+A `Task` is like assigning work to an employee. The employee handles the request (including waiting for other offices) while you continue with your immediate work.
+
+After you dispatch work to the employee, you have no means to communicate with her. You can't tell her to stop the work or know if she finished and what the result of that work was.
+
+What you actually want is to give the employee a walkie-talkie to communicate with her while she handles the request. With the walkie-talkie, you can tell her to stop, or she can tell you when she encounters an error, or she can report the result of the request you gave her.
+</div>
+
+Instead of creating unmanaged tasks, use Swift concurrency to keep control of the subtasks you create. Use `TaskGroup` to manage a (group of) subtask(s). Swift provides a couple of `withTaskGroup() { group in ... }` functions to help create a task group.
+
+```swift
+func doWork() async {
+
+    // this will return when all subtasks return, throw an error, or are cancelled
+    let result = try await withThrowingTaskGroup() { group in 
+        group.addTask {
+            try await self.performAsyncOperation1()  
+        }
+        group.addTask {
+            try await self.performAsyncOperation2()  
+        }
+        // wait for and collect the results of the tasks here 
+    }
+}
+
+func performAsyncOperation1() async throws -> Int {
+    return 1
+}
+func performAsyncOperation2() async throws -> Int {
+    return 2
+}
+```
+
+To collect the results of the group's child tasks, you can use a for-await-in loop:
+
+```swift
+var sum = 0
+for await result in group {
+    sum += result
+}
+// sum == 3 
+```
+
+If you need more control or only a few results, you can call next() directly:
+
+```swift
+guard let first = await group.next() else {
+    group.cancelAll()
+    return 0
+}
+// first == 1 or first == 2
+let second = await group.next() ?? 0
+group.cancelAll()
+// second == 2 or second == 1
+return first + second
+```
```

Comment:
For an introductory guide, you might consider removing this second code block showing `next()`. The `for await` loop above already demonstrates result collection well, and this adds complexity that might overwhelm beginners.